### ISSUE TITLE
problem to use modal

### ISSUE THREAD

**Issue by SergioAugustoPeralta** [creativetimofficial/ct-now-ui-dashboard-pro#69]
now-ui-dashboard-pro-v.1.4.1

I am trying to use modal and sow the following error:

perfect-scrollbar.jquery.min.js:6 Uncaught Error: no element is specified to initialize PerfectScrollbar
    at new H (perfect-scrollbar.jquery.min.js:6)
    at HTMLDivElement.<anonymous> (now-ui-dashboard.min.js?v=1.4.1:1)
    at HTMLDivElement.dispatch (jquery.min.js:2)
    at HTMLDivElement.y.handle (jquery.min.js:2)
    at Object.trigger (jquery.min.js:2)
    at HTMLDivElement.<anonymous> (jquery.min.js:2)
    at Function.each (jquery.min.js:2)
    at w.fn.init.each (jquery.min.js:2)
    at w.fn.init.trigger (jquery.min.js:2)
    at o.t.show (bootstrap.min.js:6)

**Comment by dragosct** [comment]
Hi, @SergioAugustoPeralta! Thank you for using our products. I just tested on the windows laptop and on our live preview everything it's working fine without any error received. You are sure you haven't made any changes in the javascript files? 

Regards, 
Dragos

**Comment by fajartaufik** [comment]
The error occurs when you rename (the modal ID)/remove once of the three modals.
![image](https://user-images.githubusercontent.com/47411329/63831515-ff7b6680-c998-11e9-84a0-83885a55bbb8.png)

I hope you can fix this.

**Comment by dragosct** [comment]
Hi, @fajartaufik! Please, can you be more specific about the issue? 

Regards, 
Dragos

**Comment by fajartaufik** [comment]
> Hi, @fajartaufik! Please, can you be more specific about the issue?
> 
> Regards,
> Dragos

Hi Dragos,
There are 3 modals in the notification page for example (noticeModal, myModal, myModal10), you can try to rename myModal with myModal1, then you refresh the page and click the notice modal button 
![image](https://user-images.githubusercontent.com/47411329/63839284-2b064d00-c9a9-11e9-916c-e0ed5240ac26.png)
![modal_nowui](https://user-images.githubusercontent.com/47411329/63839835-39089d80-c9aa-11e9-9b1a-e5b578109843.gif)

.

**Comment by dragosct** [comment]
Hi! Please be sure that you use the latest version of the product. Have you set also the new id from the modal to the `data-target` attribute from the button. 

Regards, 
Dragos

**Comment by fajartaufik** [comment]
The page I showed earlier (gif file) is a new copy of the latest version (1.4.1).
![image](https://user-images.githubusercontent.com/47411329/63924428-746e9f00-ca72-11e9-8414-034662e5033b.png)

I didn't change anything, I want to show you how that error occurred, so I changed the modal ID (one of three modal ID).

**Comment by SergioAugustoPeralta** [comment]
Open the modal it opens normal, or the problem is when will close the error that puts above.

**Comment by dragosct** [comment]
@fajartaufik have you tried to put the new id name also in the `data-target` attribute of the button that you click on it? 

Regards, 
Dragos

**Comment by fajartaufik** [comment]
Yes, I have, the issue is not in data-target attribute of the button, I think there are something wrong in your code.
![bandicam2019-08-3015-17-13-149](https://user-images.githubusercontent.com/47411329/64005281-4359a180-cb3a-11e9-9db5-0b50e4048812.gif)

Have you tried on your side what I have done?

You can reproduce the issue by removing 2 modals (myModal and myModal10) and leaving noticeModal, after that refresh the page and then click noticeModal button.
My Conclusion is, on my side, all modals should be there, you can not remove/rename them (even though I don't need it).
![image](https://user-images.githubusercontent.com/47411329/64003848-561ea700-cb37-11e9-85ef-bdfd2d302ac5.png)

**Comment by yusrenaltair** [comment]
Maybe the problem is here.
There is no length detector for this.
Thats why you can't remove the modal div from html file
![photo_2019-11-05_11-10-32](https://user-images.githubusercontent.com/56446654/68178026-f3291080-ffbc-11e9-9c3e-2f4c13fcaf96.jpg)

**Comment by fajartaufik** [comment]
Hi @young-altair, maybe that's the problem, let's @dragosct10 to investigate and fix this error.

**Comment by dragosct** [comment]
Hi, @fajartaufik! Sorry for the late response. Please see here the [video](https://www.dropbox.com/s/q8pjssyejqqi6kr/Now%20UI%20Dashboard%20PRO%20by%20Creative%20Tim%20%E2%80%93%20Google%20Chrome%202019-11-06%2010-04-49.mp4?dl=0) that I recorded for you to see that on our live preview everything is working fine without any error. Please be sure that when you upgraded your product you also imported also the js file from the new version. 

Regards, 
Dragos

**Comment by fajartaufik** [comment]
Hi @dragosct10 
please rename the modal id in your demo files (e.g id="myModal" to id="testModal")

**Comment by dragosct** [comment]
Hi! When you change the name of the modal you must go and change also in the js file the name. This is not so good and on the next update, we will fix it. 

Regards,
Dragos

**Comment by fajartaufik** [comment]
@dragosct10 
That's what I mean from the first explanation.

**Comment by dragosct** [comment]
Hi! Yes, you're right. You go in the js file and replace those ids with only the class `.modal` and then you can use any id that you want for your modals. 

![Screenshot 2019-11-07 at 11 37 57](https://user-images.githubusercontent.com/17456857/68377538-3f0dbe00-0153-11ea-83fa-ace2b82cdf9a.png)

Regards, 
Dragos

**Comment by fajartaufik** [comment]
Hi @dragosct10 ,
Perfect, thank you for your support.

**Comment by maroua2020dev** [comment]
Hello @dragosct10 i can't open modal in now ui dashboard angular

**Comment by dragosct** [comment]
Hi, @maroua2020dev! Thank you for using our products. Please use the correct repo and open the issue there, this is the HTML PRO version product. 

Regards,
Dragos

**Comment by maroua2020dev** [comment]
Hello,
Okay si my issue it's i can't add bootstrap modal and open it just by data
target..
I can't use bootstrap normally in this template and it's urgently

Thanks
